### PR TITLE
Pragmatic programmer

### PR DESCRIPTION
**Pull request by ktarbet**

### CODE REVIEW

File: HEFSConverter/DssEnsembleReader.cs [hunk 1]
```diff
@@ -86,9 +77,8 @@ public ITimeSeriesOfEnsembleLocations ReadDataset(Watersheds watershed, DateTime
       return rval;
     }
     
-    public ITimeSeriesOfEnsembleLocations ReadDatasetFromProfiles(Watersheds watershed, DateTime start, DateTime end, string dssPath)
+    public TimeSeriesOfEnsembleLocations ReadDatasetFromProfiles(Watersheds watershed, DateTime start, DateTime end, string dssPath)
```
Comment (HenryGeorgist) [inline HEFSConverter/DssEnsembleReader.cs:80]: :( sad face for tightly coupled code...

File: HEFSConverter/DssEnsembleWriter.cs [hunk 2]
```diff
@@ -12,7 +13,7 @@ namespace HEFSConverter
 {
   public class DssEnsembleWriter
   {
-    public static TimeSpan Write(string dssFileName, ITimeSeriesOfEnsembleLocations watersheds, bool saveAsFloat, int version)
+    public static TimeSpan Write(string dssFileName, TimeSeriesOfEnsembleLocations watersheds, bool saveAsFloat, int version)
```
Comment (HenryGeorgist) [inline HEFSConverter/DssEnsembleWriter.cs:16]: who needs inversion of control anyway.

**Review by HenryGeorgist — APPROVED** [review]
Very pragmatic. I am just sad to see my interfaces go.

File: HEFSConverter/DssEnsembleReader.cs (hunk 1 above)
Comment (ktarbet) [reply HEFSConverter/DssEnsembleReader.cs:80]: We can extract interface later, or use some other techniques.

File: HEFSConverter/DssEnsembleWriter.cs (hunk 2 above)
Comment (ktarbet) [reply HEFSConverter/DssEnsembleWriter.cs:16]: We should also drop the TimeSpan.  Higher level code does the timing now.

**Pull request merged**